cmd/configschema/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/fatih/structtag v1.2.0
-	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.83.0
+	github.com/google/uuid v1.3.1
+	github.com/open-telemetry/opentelemetry-collector-contrib v0.0.0-00010101000000-000000000000
```

Comment:
Is this intended?

Reply:
it is not, i had the same question here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26025#discussion_r1301816036